.github/workflows/update.yaml

```diff
@@ -0,0 +1,47 @@
+name: Update
+
+# FIXME: Delete when the PR is ready
+on:
+  push:
+    branches: 
+      - "gh-actions-update"
+
+# Enable for daily auto-updates
+# on:
+#     schedule:
+#     - cron: 07 8 * * * # 08:07 UTC is 00:07 PST
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  auto-update:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```

Comment:
Add branch `master` arg. This should always run against the `master` branch, not this branch as base.

Reply:
👍